wagtail_bucketav/models.py

```diff
@@ -7,3 +8,21 @@ class FileScanStatus(models.TextChoices):
     CLEAN = "clean", _("Clean")
     INFECTED = "infected", _("Infected")
     ERRORED = "no", _("Errored")
+
+
+class BucketAVMixin(models.Model):
+    bucketav_scan_status = models.CharField(
+        max_length=max(len(label) for label in FileScanStatus.labels),
+        choices=FileScanStatus.choices,
+        default=FileScanStatus.NOT_SCANNED,
+        verbose_name=_("bucketAV scan status"),
+    )
+    bucketav_last_scanned_at = models.DateTimeField(null=True)
+
+    def update_bucketav_scan_status(self, scan_status: FileScanStatus) -> None:
+        self.bucketav_scan_status = scan_status
+        self.bucketav_last_scanned_at = timezone.now()
+        self.save()
```

Comment:
Suggestion: Add `update_fields` here so it only updates the values we want
```suggestion
        self.save(update_fields=['bucketav_scan_status', 'bucketav_last_scanned_at'])
```

Reply:
Better indeed! Addressed in [3e9ab2a](https://github.com/torchbox/wagtail-bucketav/pull/6/commits/3e9ab2a1f72af7931b787744741041623d2150bb)